core/math/a_star_grid_2d.h

```diff
@@ -169,7 +169,7 @@ class AStarGrid2D : public RefCounted {
 	void clear();
 
 	Vector<Vector2> get_point_path(const Vector2i &p_from, const Vector2i &p_to);
```

Comment:
What about this?

Reply:
`Vector2`s are OK here, because those are the actual path positions.

I can change it to `TypedArray` as well, but this is not necessary as it works as intended.